src/storage/mod.rs

```diff
@@ -990,15 +992,34 @@ impl<E: Engine, L: LockMgr> Storage<E, L> {
                                 !ctx.get_not_fill_cache(),
                             );
                             let mut results = vec![];
-                            // TODO: optimize using seek.
+                            let use_incremental_get = gets.len() >= BATCH_INCREMENTAL_GET_LIMIT;
+                            if use_incremental_get {
+                                gets.sort_by(|a, b| match a.key.cmp(&b.key) {
```

Comment:
Do you want to sort by `key` desc? How about `gets.sort_by(|a, b| match b.key.cmp(&a.key)`